metricflow/model/transformations/proxy_measure.py

```diff
@@ -27,24 +27,24 @@ def transform_model(model: UserConfiguredModel) -> UserConfiguredModel:
 
                 add_metric = True
                 for metric in model.metrics:
-                    if metric.name == measure.name:
+                    if metric.name == measure.reference:
```

Comment:
This looks like `metric.name` hasn't been adjusted to `metric.reference`. The left intuitively is a string because `name`. The right is intuitively a reference because `reference`.

Reply:
Updated.